chore(ruby-agent): Add version tracking to errors (#17607)
newrelic/docs-website: src/content/docs/apm/agents/ruby-agent/api-guides/sending-handled-errors-new-relic.mdx

</table>

> ## Error fingerprinting: Dynamically apply an error group to each noticed error [#error-fingerprinting]

Author: Vale linter didn't like the lowercased letter following the colon

Because the API is intended to be called every time a new user ID has entered scope, it will ideally be called via middleware that is aware of user session creation. Once the New Relic Ruby agent has been made aware of the user ID, it will supply the `enduser.id` agent attribute on the current transaction as well as on any errors noticed during the current transaction's lifespan.

## Version tracking: Use metadata to see what version produced an error [#version-tracking]

[Errors inbox](https://docs.newrelic.com/docs/errors-inbox/version-tracking/) will automatically track which versions of your software are producing errors. Any version data will also display in [CodeStream](/docs/codestream/how-use-codestream/performance-monitoring/#buildsha).

> Setting one of the following environment variables will help you identify which versions of your software produce errors.

Reviewer: ```suggestion
Set one of the following environment variables to help identify which versions of your software produce errors.
```
